cmd/install_test.go

```diff
@@ -60,7 +60,7 @@ func InitGlobalConfig(libraryPath, localRepo string, update, suggests bool, inst
 	cfg = configlib.PkgrConfig{
 		Threads:  5,
 		Update:   update,
-		Rollback: false,
+		NoRollback: false,
```

Comment:
This is now doing the opposite -- is this what we want? I assume we're scrapping these tests anyway, but this could technically cause problems.

Reply:
GitHub won't let me comment on it as part of the review, but on line 296, we're still setting the "Rollback" field of cfg.

Reply:
you're right - we need to swap these - though I'm wondering if we even need these tests anymore - eg prob can/should just leverage the integration tests